payments/schema/types.py

```diff
@@ -5,7 +5,6 @@
 from leases.schema import BerthLeaseNode, WinterStorageLeaseNode
 from resources.schema import HarborNode, WinterStorageAreaNode
 from users.decorators import view_permission_required
-from utils.enum import graphene_enum
```

Comment:
do we still use this anywhere after all the changes? should we delete it?

Reply:
We're still using it in one place for:

[`resources/schema.py#L47`](https://github.com/City-of-Helsinki/berth-reservations/pull/253/files#diff-09f1721fc193a5a442fee1cf24f5bd49R47)
```python
BerthMooringTypeEnum = graphene_enum(BerthMooringType)
```